src/data/vessels.js

```diff
@@ -45,7 +45,8 @@ module.exports = source => {
           query: {
             query_string: {
               query: `*${query.query}*`,
-              allow_leading_wildcard: true
+              allow_leading_wildcard: true,
+              ...(query.queryFields && { fields: query.queryFields })
```

Comment:
https://medium.com/@mikeh91/conditionally-adding-keys-to-javascript-objects-using-spread-operators-and-short-circuit-evaluation-acf157488ede